Bottom: 06900af70bba616c5b99cee41b3b61e710c525d2
Top:    848d06fc61efdaea5bec6c9402cbf4287f1b40d4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-24 00:44:49 +0000

Refresh of find-c-alter-lookupconnection-

---

diff --git a/src/find.c b/src/find.c
index 2b7f2b4..2bb1814 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3051,8 +3051,7 @@ ListStart (int type, void *ptr1, void *ptr2, void *ptr3)
  * also the action is marked as undoable if AndDraw is true
  */
 void
-LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int which_flag,
-                  bool AndRats)
+LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int connected_flag, int found_flag)
 {
   void *ptr1, *ptr2, *ptr3;
   char *name;
